engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -57,45 +37,31 @@ object MetadataServiceActor {
   case class MetadataLookupResponse(query: MetadataQuery, eventList: Seq[MetadataEvent]) extends MetadataServiceResponse
   case class MetadataServiceKeyLookupFailed(query: MetadataQuery, reason: Throwable) extends MetadataServiceResponse
 
-  def props(serviceConfig: Config, globalConfig: Config) = {
-    Props(MetadataServiceActor(serviceConfig, globalConfig))
+  def props(dataAccess: DataAccess, serviceConfig: Config, globalConfig: Config) = {
```

Comment:
Yeah I just meant leave the implementation Chris wrote as it is, and put this one beside it, instead of overwriting it. But since HSQLDB is in memory they're pretty much equivalent I guess